hardhat.config.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-undef */
+const dotenv = require('dotenv');
+
+const dotenvResult = dotenv.config();
+
+if (dotenvResult.error) {
+  throw dotenvResult.error;
+}
+
+require('@nomiclabs/hardhat-ethers');
+require('@nomiclabs/hardhat-solhint');
+require('@nomiclabs/hardhat-truffle5');
+require('hardhat-gas-reporter');
+require('solidity-coverage');
+
+const {
+  PROVIDER_HOST,
+  PROVIDER_PORT,
+  PROVIDER_URL,
+  NETWORK,
+  MNEMONIC,
+} = process.env;
+const GWEI = 1000000000;
+
+// Ref - https://chainid.network/chains.json
+const ENV_CHAIN_IDS = {
+  mainnet: 1,
+  goerli: 1337,
+};
+
+module.exports = {
+  defaultNetwork: 'hardhat',
+  solidity: {
+    version: '0.6.11',
+    settings: {
+      optimizer: {
+        enabled: true,
+        runs: 1500,
+      },
+    },
+  },
+  paths: {
+    sources: './contracts',
+    tests: './test',
+  },
+  networks: {
+    hardhat: {
+      blockGasLimit: 0x1fffffffffffff,
+      allowUnlimitedContractSize: true,
+      chainId: 31337,
+    },
+    coverage: {
+      url: 'http://localhost:8555',
+      gas: 0xfffffffffff,
+    },
+    ganache: {
+      url: `http://${PROVIDER_HOST}:${PROVIDER_PORT}`,
+      network_id: 31337,
+      logger: console,
+    },
+    // More about networks config:
+    // https://hardhat.org/config/#json-rpc-based-networks
+    goerli: {
+      url: PROVIDER_URL || '',
+      accounts: { mnemonic: MNEMONIC },
+      chainId: ENV_CHAIN_IDS[NETWORK],
+      gas: 7700000,
+      gasPrice: 1 * GWEI,
+    },
+  },
+  gasReporter: {
+    outputFile: 'gas-report.txt',
```

Comment:
The default of outputing to the stdout should be fine IMO

Reply:
Refs-: https://github.com/razor-network/contracts/pull/77#discussion_r581884746